### PR TITLE
Async/Await iOS 13 Support

### PR DESCRIPTION
**Pull request by laqiguy**
Added iOS 13 support for async/await with URLSession extension.

### CODE REVIEW

**Review by Alexander-Ignition — APPROVED** [review]

File: Sources/ApexyAlamofire/AlamofireClient.swift [hunk 1]
```diff
@@ -5,6 +5,7 @@
 //  Copyright © 2019 RedMadRobot. All rights reserved.
 //
 
+import Apexy
```
Comment (subdan) [inline Sources/ApexyAlamofire/AlamofireClient.swift:8]: Is it necessary?

File: Sources/ApexyURLSession/URLSessionClient.swift [hunk 2]
```diff
@@ -156,7 +157,7 @@ open class URLSessionClient: Client {
         }
     }
     
-    @available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
+    @available(macOS 10.15, iOS 12, watchOS 6, tvOS 13, *)
```
Comment (subdan) [inline Sources/ApexyURLSession/URLSessionClient.swift]: iOS 12 or 13?

File: Sources/ApexyAlamofire/AlamofireClient.swift [hunk 3]
```diff
@@ -192,7 +193,7 @@ open class AlamofireClient: Client {
         })
     }
     
-    @available(macOS 12, iOS 15, watchOS 8, tvOS 15, *)
+    @available(macOS 10.15, iOS 12, watchOS 6, tvOS 13, *)
```
Comment (subdan) [inline Sources/ApexyAlamofire/AlamofireClient.swift]: Same here: iOS 12 or 13?

File: Sources/ApexyAlamofire/AlamofireClient.swift (hunk 1 above)
Comment (laqiguy) [reply Sources/ApexyAlamofire/AlamofireClient.swift:8]: You can check - there is `Cannot find type '' in scope` error everywhere, if `import Apexy` is not added

**Review by vani2 — APPROVED** [review]

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -14,9 +14,11 @@ on:
 jobs:
   test:
     name: Run tests
-    runs-on: macOS-latest
+    runs-on: macos-11
     steps:
       - name: Checkout
         uses: actions/checkout@v2
+      - name: XCode select
```
Comment (subdan) [inline .github/workflows/tests.yml]:
```suggestion
      - name: Change Xcode
```

Comment (subdan) [reply .github/workflows/tests.yml]: Do we need to change Xcode version for release workflow?

**Review by subdan — APPROVED** [review]

**Pull request merged**